hydrolib/core/dflowfm/ext/models.py

```diff
@@ -254,7 +255,9 @@ class Comments(INIBasedModel.Comments):
 
     _header: Literal["Meteo"] = "Meteo"
     quantity: str = Field(alias="quantity")
-    forcingfile: ForcingModel = Field(alias="forcingFile")
+    forcingfile: Union[ForcingModel, TimModel, DiskOnlyFileModel] = Field(
```

Comment:
At first sight, this looks good, but unfortunately a unit test with a `.tim` file fails. See my recent commit.

Reply:
The provided tim file was incorrect, so pydantic decides that it is probably not a `TimModel`, but the next one that does not fail: `ForcingModel`. The tim file cannot contain duplicate times and text after the columns.
I also switched the order of the union, as `ForcingModel` seems to be less strict.

Reply:
ahh, apologies for causing confusion!